app/models/discourse_post_event/event.rb

```diff
@@ -53,12 +53,14 @@ def set_next_date
         end
       end
 
-      publish_update!
-      invitees.update_all(status: nil, notified: false)
+      invitees.where.not(status: Invitee.statuses[:going]).update_all(status: nil, notified: false)
```

Comment:
This is the main change. Instead of selectioning every invitees we filter out the going invitees

Reply:
I am wondering if there should be a "going to this event" or "going to all event options" like other calendars have.

Reply:
maybe, but for now we will assume it's by default "going to this and all future events"